Remove unqueryable lists from admin meta (#6512)
keystonejs/keystone: .changeset/twelve-apples-happen.md

---
'@keystone-next/keystone': patch
---

> Fixed unqueryable lists being accessible via the Admin UI.

Reviewer: ```suggestion
Fixed lists with `graphql.omit: ['query']` causing issues in the Admin UI.
```
